### ISSUE TITLE
syntax error loading ppx_deriving.std in rtop

### ISSUE THREAD

**Issue by freebroccolo** [facebook/reason#739]
Is there a way to get `ppx_deriving` to work in rtop? When I try to load `ppx_deriving.std` I just get a syntax error:

```
Reason # #require "ppx_deriving.std";
Error: Syntax error
Reason #
```

**Comment by jordwalke** [comment]
What's interesting is that the following does not cause a syntax error:

```
 #require "ppx_deriving.asdf";
```

I wonder if `ppx_deriving.std` is the code throwing the syntax error? Does everything compile with `ppx_deriving.std` when not used in the top level?

**Comment by freebroccolo** [comment]
Yeah, it seems to work okay if you just do `#require "ppx_deriving";` too.

> I wonder if ppx_deriving.std is the code throwing the syntax error? Does everything compile with ppx_deriving.std when not used in the top level?

Yeah it works fine if I just compile to a binary and run that way.

**Comment by freebroccolo** [comment]
I should clarify that `#require "ppx_deriving";` doesn't throw an error, but in fact `#require "ppx_deriving.std";` is necessary to load the library I need since it contains the drivers.

Another thing is that if I compile using reason syntax with `rebuild` but load `utop` instead everything works okay. So it's definitely something about the interaction between `ppx_deriving` and `rtop`.

**Comment by jordwalke** [comment]
I see, okay thanks.
